### DISCUSSION TITLE
Game doesn't run when I click on Play

### DISCUSSION THREAD

**Discussion by BootsManOut** [PhaserEditor2D/PhaserEditor2D-v3#159] [Q&A]
Hi there!

I just installed Phase Editor 2D.
I loaded the platformer example template and selected a folder for the project.
I'm running it in Windows 10. I can see the assets and scenes in the editor, but when I click the 'Play' button to run the game, absolutely nothing happens.
And I don't have any menu bar, where I could access the 'Preferences' and set a standard browser. All I have is the three line drop button from where you can open projects, select color theme etc., but I cannot access the 'Preferences' from there.

What am I supposed to do here?

**Comment by PhaserEditor2D** [comment]
Hi!

Please, can you check the DevTools of the browser showing the game and see if there is any error there?

**Reply by BootsManOut** [reply]
Hey! Well, I just started following the tutorial. How do I go about running it in the browser?

**Reply by PhaserEditor2D** [reply]
Hi. Please, try the latest version 3.31.0. It launches the browser when you click in the "play" button.

**Reply by BootsManOut** [reply]
Okay thanks!
